independent-projects/resteasy-reactive/common/runtime/pom.xml

```diff
@@ -22,6 +22,14 @@
             <groupId>io.smallrye.reactive</groupId>
             <artifactId>mutiny</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.reactivestreams</groupId>
+            <artifactId>reactive-streams</artifactId>
```

Comment:
Why do we add that dependency?

Reply:
We need to add it because:

1. Mutiny does not depend on RS anymore, and
2. We are in a transition period where RestEasy Reactive will support both legacy RS APIs and JDK `Flow`.

That being said `mutiny-zero-flow-adapters` does bring that dependency, but I thought it'd be better to be explicit about it, WDYT?

Reply:
Yes, if we want to support both, it's probably better to be explicit. I thought we would drop that support entirely in Quarkus 3.

Reply:
It's really safer to have Quarkus 3 supporting both APIs, and then we can check what's the ecosystem state in v4